contracts/compound-rate-keeper/AbstractCompoundRateKeeper.sol

```diff
@@ -96,7 +93,8 @@ abstract contract AbstractCompoundRateKeeper is ICompoundRateKeeper, Initializab
         uint256 rate_ = _currentRate;
 
         if (capitalizationPeriodsNum_ != 0) {
-            uint256 capitalizationPeriodRate_ = capitalizationRate_.rpow(
+            uint256 capitalizationPeriodRate_ = _rpow(
```

Comment:
May possibly overflow, because `capitalizationRate_` is set by user. Can we return `_getMaxRate()` in view functions in that case?